oozie-to-airflow/mappers/base_mapper.py

```diff
@@ -69,6 +69,20 @@ def last_task_id(self) -> str:
         """
         return self.name
 
+    def on_parse_node(self, workflow):
+        """
+        Called when processing a node.
+
+        This is a good time to read all data from a node.
+        """
+
+    def on_finish_parse_nodes(self, workflow):
```

Comment:
Okay I just noticed it is being using in the next PR https://github.com/GoogleCloudPlatform/cloud-composer/pull/68.
For the future it would be good to move this method to the PR where it is actually used or, if it's too difficult, to at least clearly indicate this in the PR description.

Reply:
What should contain PR is a long and complex topic. I think that the current division is correct.

Reply:
Well, I'm telling you that for me, as a reviewer, it's unclear why this method is here if it's not used anywhere. So it's creating confusion and it should at the very least be explained.

Reply:
I agree with @sprzedwojski  fully that adding change that will be only used in the next PR is bad practice. Splitting the PRs into logical chunks that can be read and understood separately makes perfect sense.